src/sentry/integrations/github/webhook.py

```diff
@@ -183,6 +192,7 @@ def _handle(
                 continue
 
     def __call__(self, event: Mapping[str, Any], **kwargs: Any) -> None:
+        github_event = kwargs["github_event"]
```

Comment:
We pass this as a `kwargs` since we don't want to touch the signature of the base class.